### PR TITLE
chore: Fix linter findings for `revive:exported` in `plugins/inputs/d*`

### PR DESCRIPTION
**Pull request by zak-pawel**
## Summary

Address findings for [revive:exported ](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#exported ) in `plugins/inputs/d*`.

As part of this effort for files from `plugins/inputs/d*`, the following actions were taken:
- Type names (`const`, `var`, `struct`, `func`, etc) were changed to unexported, wherever they didn't need to be exported.
- All remaining exported types were given comments in the appropriate form – this does not apply to exported methods that implement "known" plugin interfaces (`Gather|Init|Start|Stop|SampleConfig|Parse|Add|Apply|Serialize|SerializeBatch|SetParser|SetParserFunc|GetState|SetState`).
- The order of methods was organized (exported methods first, then unexported, with `init` at the very end).

It is only part of the bigger job (for issue: https://github.com/influxdata/telegraf/issues/15813).
After all findings of this type in whole project are handled, we can enable `revive:exported` rule in `golangci-lint`.

## Checklist

- [x] No AI generated code was used in this PR

### CODE REVIEW

**Review by DStrand1 — APPROVED** [review]

File: plugins/common/docker/stats_helpers.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/docker/docker/api/types/container"
 )
 
+// CalculateCPUPercentUnix calculate CPU usage (for Unix, in percentages)
 func CalculateCPUPercentUnix(previousCPU, previousSystem uint64, v *container.StatsResponse) float64 {
```
Comment (srebhan) [inline plugins/common/docker/stats_helpers.go:10]: Do we really need all those exported?

File: plugins/inputs/directory_monitor/directory_monitor.go [hunk 2]
```diff
@@ -31,11 +31,13 @@ import (
...
 var sampleConfig string
 
-var once sync.Once
-
 var (
-	defaultFilesToMonitor             = []string{}
-	defaultFilesToIgnore              = []string{}
+	once                  sync.Once
+	defaultFilesToMonitor = []string{}
+	defaultFilesToIgnore  = []string{}
```
Comment (srebhan) [inline plugins/inputs/directory_monitor/directory_monitor.go:37]: Do we really need those?

**Review by srebhan — COMMENTED** [review]
Thanks @zak-pawel! Just two comments...

File: plugins/common/docker/stats_helpers.go (hunk 1 above)
Comment (zak-pawel) [reply plugins/common/docker/stats_helpers.go:10]: These methods are used by two plugins (`docker` and `ecs`). 
It seems reasonable to me that they ended up in the `common` packages - this way one plugin (`ecs`) doesn't import another plugin (`docker`).

Comment (srebhan) [reply plugins/common/docker/stats_helpers.go:10]: Oh boy I completely missed the `common` part here... Then of course this needs to be exported. Sorry for the noise!

File: plugins/inputs/directory_monitor/directory_monitor.go (hunk 2 above)
Comment (zak-pawel) [reply plugins/inputs/directory_monitor/directory_monitor.go]: I believe we don't

**Review by srebhan — APPROVED** [review]
Thanks @zak-pawel!

**Pull request merged**